### PR TITLE
Inject css for adding a hover effect to edges

### LINKED ISSUE #13: Highlight edges when hovering over them
For the const eval skill tree I added

```html
<style>
      /* the lines within the edges */
      .edge:active path,
      .edge:hover path {
        stroke: fuchsia;
        stroke-width: 3;
        stroke-opacity: 1;
      }
      /* arrows are typically drawn with a polygon */
      .edge:active polygon,
      .edge:hover polygon {
        stroke: fuchsia;
        stroke-width: 3;
        fill: fuchsia;
        stroke-opacity: 1;
        fill-opacity: 1;
      }
      /* If you happen to have text and want to color that as well... */
      .edge:active text,
      .edge:hover text {
        fill: fuchsia;
      }
</style>
```

to the mdbook contents in order to make edges get highlighted (thick and pink) when you hover over them or their arrow heads. It may be useful to have `mdbook-skill-tree` automatically generate this.

### PR DESCRIPTION
**Pull request by oli-obk**
fixes #13

### CODE REVIEW

**Comment by nikomatsakis** [comment]
My suggestion was going to be to modify this function:

https://github.com/nikomatsakis/skill-tree/blob/7e5cb451cda105cc20207311da3fe9dc8d5dd4d9/mdbook-skill-tree/src/main.rs#L176-L196

to include something like

```rust
    for file in CSS_FILES {
        let additional_css = additional(doc, "css");
        if has_file(&additional_css , file.name) {
            log::debug!("'{}' already in 'additional-css'. Skipping", file.name)
        } else {
            if !printed {
                printed = true;
                log::info!("Adding additional files to configuration");
            }
            log::debug!("Adding '{}' to 'additional-css'", file.name);
            insert_additional(doc, "css", file.name);
            changed = true;
        }
    }
```

In that case, I don't think we need the `<script>` business, and I think mdbook will package up the css file appropriately. It does mean you have to run `mdbook-skill-tree install` but that was already true.

**Comment by oli-obk** [comment]
Oh, I assumed from https://github.com/nikomatsakis/skill-tree/issues/13#issuecomment-651955216 that wouldn't work, but since we're modifying the book during the install, we can do this :facepalm:

**Comment by oli-obk** [comment]
Fixed

File: mdbook-skill-tree/src/main.rs [hunk 1]
```diff
@@ -177,17 +186,17 @@ fn add_additional_files(doc: &mut Document) -> bool {
     let mut changed = false;
     let mut printed = true;
 
-    for file in JS_FILES {
-        let additional_js = additional(doc, "js");
-        if has_file(&additional_js, file.name) {
-            log::debug!("'{}' already in 'additional-js'. Skipping", file.name)
+    for file in ADDITIONAL_FILES {
```
Comment (nikomatsakis) [inline mdbook-skill-tree/src/main.rs:189]: beautiful

**Comment by nikomatsakis** [comment]
CI is grumpy but it looks spurious

**Comment by oli-obk** [comment]
I tested it by installing it on the `book` in this repo, that worked just fine.

**Pull request merged**